sites/vision-systems/server/routes/content.js

```diff
@@ -1,18 +1,13 @@
 const { withContent } = require('@base-cms/marko-web/middleware');
 const company = require('../templates/content/company');
 const content = require('../templates/content');
-const whitepaper = require('../templates/content/whitepaper');
 const queryFragment = require('../graphql/fragments/content-page');
 
 module.exports = (app) => {
   app.get('/*?company/:id(\\d{8})*', withContent({
     template: company,
     queryFragment,
   }));
-  app.get('/*?whitepaper/:id(\\d{8})*', withContent({
```

Comment:
Why is the whitepaper _content_ page being removed?

Reply:
I removed it because I did not believe it would be used anymore. If it should be added back in, please let me know.

Reply:
Yes the white paper content page needs to stay. That’s what displays the custom layout when you view an individual white paper.